Development.md

```diff
@@ -148,6 +148,12 @@ spec:
 EOF
 ```
 
+Add `istiod.istio-system.svc` to the system's `hosts` file so that [`verify_is_valid_for_subject_name`](https://crates.io/crates/rustls-webpki) passes:
+
+```shell
+echo "127.0.0.1 istiod.istio-system.svc" >> /etc/hosts
```

Comment:
```suggestion
echo "127.0.0.1 istiod.istio-system.svc" >> /etc/hosts
```

We shouldn't advise people to muck with their local `hosts` file - we shouldn't have any tests that depend on this.

Reply:
you right.. we maybe add some flag to skip dns subject check.